api/src/main/resources/liquibase.xml

```diff
@@ -585,6 +585,11 @@
 			<column name="multiset_name" type="varchar(255)" />
 		</addColumn>
 	</changeSet>
+	
```

Comment:
Liquibase file cannot be changed retrospectively. Please add a new change set at the end, and with the right date in ID.